src/Symfony/Component/Mime/RawMessage.php

```diff
@@ -20,7 +20,7 @@ class RawMessage
 {
     private $message;
 
-    public function __construct(iterable|string $message)
+    public function __construct(iterable|string|null $message)
```

Comment:
🔴  Allowing `null` there will throw a fatal `TypeError` ligne 37 when calling `toString()` or `__serialize()`

In your case, instead of `null` maybe try to use an empty array ?